state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1506,14 +1506,13 @@ pub mod pallet {
 					"All swaps should have Stable amount set here"
 				);
 
-				let mut stable_amount = swap.stable_amount.unwrap_or_default();
-				swap.stable_amount_before_fees = Some(stable_amount);
```

Comment:
This is actually subtly different to before. If there are no fees in `swap.swap.fees`, and swap.stable amount is set to None then `stable_amount_before_fees` might stay as None.

In the previous implementation this was guaranteed to be Some after a call to this function.